app/html/sideNav/sideNavDiscovery.js

```
@@ -110,12 +114,12 @@ exports.create = (api) => {
           notifications: notifications(feedId),
           imageEl: api.about.html.avatar(feedId, 'small'),
           label: api.about.obs.name(feedId),
-          selected: location.feed === feedId && !isDiscoverLocation(location),
+          selected: get(location, 'participants', []).join() === feedId && !isDiscoverLocation(location),
```

@mixmix just a comment and a request for advise: would it be better to switch the parts of this boolean operation like: 

```
!isDiscoverLocation(location) && get(location, 'participants', []).join() === feedId,
```
Which would fail faster and avoid the `get` + `join` computation for all cases which are not discovery?

---

Also, should we adopt a style where we make the first parts of our boolean operations be the least intensive ones?